cookbooks/resources_allocation.rst

```diff
@@ -0,0 +1,176 @@
+Resources allocation
+====================
+
+SymfonyCloud handles resource allocation in a reserved approach, rather than
+on-demand. This means that your project (applications plus services) are given a
+certain amount of resources and can use as much, or as little, of them as
+needed.
+
+Plan size
+---------
+
+Your production plan size determines how much CPU and RAM are available to your
+entire production environment.
+
+.. note::
+
+   The development plan has no dedicated resources allocated: the main
+   environment is treated like a
+   :ref:`development environment <dev-env-sizing>` and container sizes are set
+   to ``S``.
+
+You can change your project plan at anytime using the
+``symfony project:scale <plan>`` command.
+
+Possible production plan sizes are:
+
+* Standard
```

Comment:
When running `symfony project:scale`, the arg is the plan name like l, m, ..., right? If that's the case, we should probably add them to the list below

Reply:
The optional arg is one:
- `development`/`dev`
- `standard`
- `medium`
- `large`
- `xlarge`
- `2xlarge`